src/components/NavigationSection/__tests__/__snapshots__/index.test.tsx.snap

```diff
@@ -5,10 +5,10 @@ exports[`NavigationSection component renders correctly 1`] = `
   <div
     class="navigationSection"
   >
-    <div
+    <button
```

Comment:
I think originally I made them `divs` as I didn't know precisely what `aria-` tags to use correctly. Thank you!